src/panels/lovelace/common/create-entity-row-element.js

```diff
@@ -17,12 +41,7 @@ function _createElement(tag, config, stateObj, hass) {
     return _createErrorElement(err.message, config);
   }
 
-  element.stateObj = stateObj;
   element.hass = hass;
```

Comment:
Do we still need this or can we start removing `hass` from this file ? (as it should be)